base/client.jl

```diff
@@ -292,12 +292,12 @@ function exec_options(opts)
             invokelatest(show, Core.eval(Main, parse_input_line(arg)))
             println()
         elseif cmd == 'm'
-            @eval Main import $(Symbol(arg)).main
+            entrypoint = push!(split(arg, "."), "main")
+            Base.eval(Main, Expr(:import, Expr(:., Symbol.(entrypoint)...)))
```

Comment:
I could implement this without `eval` by using `Base.identify_package` + `Base.require` + traversing the module returned from `require`. Is that worth it?

Reply:
> Is that worth it?

No